cni/network/plugin/main.go

```diff
@@ -289,17 +293,21 @@ func main() {
 		os.Exit(0)
 	}
 
-	log.SetName(name)
-	log.SetLevel(log.LevelInfo)
-	if err := log.SetTargetLogDirectory(log.TargetLogfile, ""); err != nil {
+	loggerCfg := &log.Config{
+		Level:       zapcore.DebugLevel,
+		LogPath:     log.LogPath + "azure-vnet.log",
```

Comment:
can it be `LogPath: log.LogPath + name + ".log"`

Reply:
done